resources/views/manager/job_index/job.html.twig

```diff
@@ -6,153 +6,131 @@
-            {% if job.created_at %}<span class="poster-date-time--draft">{{ job.classification }} - {{ jobs_l10n.created_on }} {{ humanizeDate(job.created_at) }}</span>{% endif %}
-            {% if job.review_requested_at %}<span class="poster-date-time--submitted">{{ job.classification }} - {{ jobs_l10n.submitted_on }} {{ humanizeDate(job.review_requested_at) }}</span>{% endif %}
-            {% if job.published_at %}<span class="poster-date-time--published">{{ job.classification }} - {{ jobs_l10n.published_on }} {{ humanizeDate(job.published_at) }}</span>{% endif %}
-            {% if job.close_date_time %}<span class="poster-date-time--closed">{{ job.classification }} - {{ jobs_l10n.closed_on }} {{ humanizeDate(job.close_date_time) }}</span>{% endif %}
-        </div>
-
-        <div class="box med-2of12 poster-measurements">
-
-            {# Note: The "measurement--comments element is removed from the UI until commenting is built. There is nothing to do with this element until then. #}
-            {#<div class="measurement--comments">
+	<div class="flex-grid middle gutter">
+
+		<div class="box med-1of12 poster-status">
+			<i class="fas fa-pencil-alt"></i>
+			<i class="fas fa-hourglass-half"></i>
+			<i class="fas fa-exclamation-circle action-required"></i>
+		</div>
+
+		<div class="box med-4of12">
+
+			{% if job.status() == 'draft' or job.status() == 'submitted' %}
+				<h3 class="heading--04 poster-title--preview">
+					<a href="{{ job.preview_link }}" title="{{ jobs_l10n.view_preview_job_poster }}">
+						{{ job.trans_required ? jobs_l10n.trans_title_placeholder ~ ' - ' }}
+						{{ not job.title ? jobs_l10n.empty_title ~ ' - ' }}
+						{{ job.title }}
+						{{ jobs_l10n.preview }}
+					</a>
+				</h3>
+			{% endif %}
+
+			{% if job.status() == 'published' or job.status() == 'closed' %}
+				<h3 class="heading--04 poster-title--link">
+					<a href="{{ route('manager.jobs.show', job) }}" title="{{ jobs_l10n.view_this_job_poster }}">
+						{{ job.title }}
+					</a>
+				</h3>
+			{% endif %}
+			{% if job.created_at %}
+				<span class="poster-date-time--draft">{{ job.classification.key }}-{{ job.classification_level }} - {{ jobs_l10n.created_on }}
+					{{ humanizeDate(job.created_at) }}</span>
+			{% endif %}
+			{% if job.review_requested_at %}
+				<span class="poster-date-time--submitted">{{ job.classification.key }}-{{ job.classification_level }} - {{ jobs_l10n.submitted_on }}
+					{{ humanizeDate(job.review_requested_at) }}</span>
+			{% endif %}
+			{% if job.published_at %}
+				<span class="poster-date-time--published">{{ job.classification.key }}-{{ job.classification_level }} - {{ jobs_l10n.published_on }}
+					{{ humanizeDate(job.published_at) }}</span>
+			{% endif %}
+			{% if job.close_date_time %}
+				<span class="poster-date-time--closed">{{ job.classification.key }}-{{ job.classification_level }} - {{ jobs_l10n.closed_on }}
+					{{ humanizeDate(job.close_date_time) }}</span>
+			{% endif %}
```

Comment:
Same here with `classification_level`, we should check if it should be left padded with a zero. Display on this should be consistent across the site if possible, probably with a mutator on the Classification model itself.